### PR TITLE
fix: add flag to control boundary init container creation

### PR DESCRIPTION
**Pull request by msalman899**

### CODE REVIEW

File: charts/boundary-chart/values.yaml [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 replicaCount: 1
 
+initialise_boundary:
```
Comment (janikgar) [inline charts/boundary-chart/values.yaml]: The change looks good - any opposition to using camelCase here?
E.g. `initialiseBoundary` ?

**Review by janikgar — APPROVED** [review]
Made a comment on formatting, but otherwise lgtm

File: charts/boundary-chart/values.yaml (hunk 1 above)
Comment (msalman899) [reply charts/boundary-chart/values.yaml]: Not at all, just pushed change.

Comment (msalman899) [reply charts/boundary-chart/values.yaml]: Please merge to master if all looks good to you.

**Pull request merged**